nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -300,6 +300,7 @@ let
             bindsTo = optional (!config.boot.isContainer) "dev-net-tun.device";
             after = optional (!config.boot.isContainer) "dev-net-tun.device" ++ [ "network-pre.target" ];
             wantedBy = [
+              "network.target"
               "network-setup.service"
```

Comment:
Is network-setup not part of network.target? If it is: would it be risky to replace it?

Reply:
> Is network-setup not part of network.target? If it is: would it be risky to replace it?

It's not: it `wants` network.target, but is `wantedBy` multi-user.target.